printer_mods/cmrny/v01_motor_panel_no_rub/README.md

```diff
@@ -0,0 +1,17 @@
+# Voron v0.1
+![Voron v0.1 Motor Panel Modification No Rub <](images/motor_panel_no_rub.png)
+
+## What is this?
+This modification keeps the umbilical and bowtub from rubbing against the motor belts.
```

Comment:
Typo. "bowtub" should be "reverse bowden tube"

Reply:
Will do :D